packages/hooks/src/index.ts

```diff
@@ -45,3 +45,5 @@ export { default as useTotalAccountBalance } from './useTotalAccountBalance';
 export { default as useVerifyEmail } from './useVerifyEmail';
 export { default as useP2PAdvertList } from './useP2PAdvertList';
 export { useIsAccountStatusPresent } from './useIsAccountStatusPresent';
```

Comment:
```suggestion
export { default as useIsAccountStatusPresent } from './useIsAccountStatusPresent';
```
Out of this PR scope but can you update the `useIsAccountStatusPresent` hook to follow the same syntax as other files 🙏🏻 Not sure how it passed code review before 👀